pysndfx/dsp.py

```diff
@@ -278,11 +332,15 @@ def vol(self, gain, type="amplitude", limiter_gain=None):
         if type in ["amplitude", "power", "dB"]:
             self.command.append(type)
         else:
-            raise InvalidEffectParameter("Type has to be dB, amplitude or power")
+            raise ValueError("Type has to be dB, amplitude or power")
         if limiter_gain is not None:
             self.command.append(str(limiter_gain))
         return self
 
+    def custom(self, command_str):
```

Comment:
Could you add a docstring to this method with a short example of a valid command_str?

Reply:
And a corresponding command here: https://github.com/carlthome/python-audio-effects/blob/master/tests/test_dsp.py#L11

Reply:
Ok.

Regarding the tests, you want me to add a unit test for this command?

Reply:
Yes, please.